newbank/server/NewBank.java

```diff
@@ -33,7 +74,7 @@ private void addTestData() {
         kim.addAccount(new Account(generateAccountNumber(), "Checking", 1050.0));
         customers.put("Kim", kim);
 
-        Customer andy = new Customer("password");
+        Customer andy = new Customer(SecurityUtilities.getSecurePassword("password"));
```

Comment:
Good job making Andy's password so secure. I would suggest that if you have already done such a good job, you should also add this level of security to the other accounts ;-).